.github/workflows/dockerfile_sanity.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Test Docker Image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'Dockerfile.intel'
```

Comment:
docker/Dockerfile.intel

Reply:
Thx! I used workflow_dispatch to manually trigger and test the workflow and forgot to update path for the auto triggering with file changes. Added!